Docker image build fails

The `docker build` command fails with the following error

```
Sending build context to Docker daemon   9.82MB
Step 1/7 : FROM node:latest
 ---> 2698faaff1ee
Step 2/7 : ENV NODE_ENV=production
 ---> Using cache
 ---> 997bbc091e8f
Step 3/7 : ENV ELASTIC_APM_JS_BASE_SERVICE_NAME=opbeans-react
 ---> Using cache
 ---> b694f0aef784
Step 4/7 : WORKDIR /app
 ---> Using cache
 ---> 4c444aa3cb59
Step 5/7 : ADD . /app
 ---> 7781db37928c
Step 6/7 : RUN npm install && npm run-script build && rm -rf node_modules
 ---> Running in 927778f3b083
npm WARN lifecycle client@1.0.0~preinstall: cannot run in wd client@1.0.0 if ! [ -d elastic-apm-rum ]; then git clone https://github.com/elastic/apm-agent-js-base elastic-apm-rum; fi (wd=/app)
npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update! 
npm WARN deprecated istanbul@0.4.5: This module is no longer maintained, try this instead:
npm WARN deprecated   npm i nyc
npm WARN deprecated Visit https://istanbul.js.org/integrations for other alternatives.

> uglifyjs-webpack-plugin@0.4.6 postinstall /app/node_modules/webpack/node_modules/uglifyjs-webpack-plugin
> node lib/post_install.js

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN css-loader@1.0.1 requires a peer of webpack@^4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-react-app@0.5.2 requires a peer of babel-eslint@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-react-app@0.5.2 requires a peer of eslint@^3.8.1 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-jsx-a11y@2.2.3 requires a peer of eslint@^2.10.2 || 3.x but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-react@6.4.1 requires a peer of eslint@^2.0.0 || ^3.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN extract-text-webpack-plugin@2.1.2 requires a peer of webpack@^2.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN html-webpack-plugin@2.24.0 requires a peer of webpack@1 || ^2.1.0-beta but none is installed. You must install peer dependencies yourself.
npm WARN webpack-dev-server@1.16.2 requires a peer of webpack@>=1.3.0 <3 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 1321 packages from 738 contributors and audited 10912 packages in 32.967s
found 8 vulnerabilities (4 low, 1 moderate, 2 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details

> client@1.0.0 build /app
> node scripts/build.js

Creating an optimized production build...
Failed to compile.

Module not found: Error: Can't resolve 'elastic-apm-js-core' in '/app/src'

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! client@1.0.0 build: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the client@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-03-26T16_35_15_211Z-debug.log
The command '/bin/sh -c npm install && npm run-script build && rm -rf node_modules' returned a non-zero code: 1
```

--- a/src/routes.js
+++ b/src/routes.js
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Router, Route, IndexRedirect } from 'react-router';
-import { apm } from 'elastic-apm-js-base/src/index'
+import { apm } from '@elastic/apm-rum'
 
 import App from './components/App';
 import Dashboard from './components/Dashboard';

--- a/src/rum.js
+++ b/src/rum.js
@@ -1,7 +1,7 @@
 
-import { init as initApm } from 'elastic-apm-js-base/src/index'
+import { init as initApm } from '@elastic/apm-rum'
 import { browserHistory } from 'react-router'
-import { patching } from 'elastic-apm-js-core'
+import { patching } from '@elastic/apm-rum-core'
 
 function changeRoute() {
     var links = Array.from(document.getElementsByTagName('a')).map((a) => {

